doc-content/enterprise-only/openshift/template-overview-con.adoc

```diff
@@ -0,0 +1,52 @@
+[id='template-overview-con']
+= Template reference information
+
+The following templates are provided with the product. You can extract the templates from the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file. You can download the file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page. 
+
+ifdef::PAM[]
+ifeval::["{context}"=="openshift-authoring"]
+* `rhpam71-authoring.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. The {KIE_SERVER} uses an H2 database with persistent storage. You can use this environment to author processes, services, and other business assets. For details about this template, see <<rhpam71-authoring-ref>>.
+
+* `rhpam71-authoring-ha.yaml` provides a high-availability {CENTRAL}, a {KIE_SERVER} connected to the {CENTRAL}, and a MySQL instance that the {KIE_SERVER} uses. You can use this environment to author processes, services, and other business assets. The high-availability functionality is in technical preview. For details about this template, see <<rhpam71-authoring-ha-ref>>.
+endif::[]
+
+ifeval::["{context}"=="openshift-managed"]
+* `rhpam71-prod.yaml` provides a high-availability {CENTRAL} Monitoring instance, a Smart Router, two distinct {KIE_SERVERS} connected to the {CENTRAL} and to the Smart Router, and two PostgreSQL instances; each {KIE_SERVER} uses its own PostgreSQL instance. You can use this environment to execute business assets in a production or staging environment. For details about this template, see <<rhpam71-prod-ref>>.
```

Comment:
"instances. Each {KIE_SERVER} uses..." Comes up a few times. I don't think it's a necessary or called-for use of the semicolon.